xwiki-rendering-integration-tests/src/test/resources/simple/macros/macro20.test

```diff
@@ -37,7 +37,7 @@ onWord [paragraph]
 onSpecialSymbol [.]
 endParagraph
 onMacroStandalone [testcontentmacro] [] [{{testcontentmacro}}
-<p><strong>New content</strong></p>
+**New content**
```

Comment:
So you did had a test after all, was just not expecting the right result :)

Reply:
yep

Reply:
Actually it's wrong, the input is `.input|xhtml/1.0` so it should remain  as it was: here it changes because I forced the target syntax to xwiki/2.0

Reply:
So inhering parent node was not working properly and you fixed another bug ?

Reply:
No it's just that the targetSyntax here should be xhtml, so the result should remain:
```
<p><strong>New content</strong></p>
```

Here the change is coming because I was forcing the targetSyntax as `xwiki/2.0` and I was not using what's defined in the test.
Now as I'm putting the RenderingContext elsewhere I'm using the targetSyntax using info from the test, so not forcing it anymore.

Reply:
I see so that test was actually right and this change need to be reverted (and a new test added).

Reply:
I need to be reverted. Now I don't know how to write the test we want: how to express using annotatedxhtml that I'm in the context of a page using xwiki/2.0 in our rendering test framework?

Reply:
It's simple a test which convert annotatedhtml (input) content into xwiki/2.1 (expect) content. In this use case the target syntax will be xwiki/2.1 with the change to are making to the rendering framework.